src/zoo/owl_zoo_cmd.ml

```diff
@@ -21,6 +21,7 @@ let preprocess script =
     "#use \"topfind\"\n" ^
     "#require \"owl\"\n" ^
     "#require \"owl-zoo\"\n" ^
+    "#require \"owl-plplot\"\n" ^
```

Comment:
This change breaks the design of zoo. Currently `owl-plplot` is a separate library, and requiring all the zoo script to rely on this external library is not a good choice.  If a user needs plotting function, he should require this library in a zoo script only.